db/schema.ts

```diff
@@ -48,17 +49,25 @@ export const projects = pgTable(
   (table) => [primaryKey({ columns: [table.id, table.name, table.userId] })],
 );
 
-// Schema for inserting a project - can be used to validate API requests
+// Schema for inserting a project - can be used to validate API requests.
+// Reach hook form doesn't like nulls, so we exclude them from the schema.
 export const insertProjectSchema = createInsertSchema(projects, {
```

Comment:
drizzle-zod methods like `createInsertSchema` end up creating nullable fields for optional columsn. This is a problem when using the schema with react hook form, which doesn't work with null.
As a result, we use `extend` to overwrite the nullable fields (making them `optional` instead, which uses undefined).